docs/contributors.rst

```diff
@@ -0,0 +1,19 @@
+Contributors
+============
+
+``pyhf`` is openly developed and benefits from the contributions and feedback
+from its users.
+The ``pyhf`` dev team would like to thank all contributors to the project for
+their support and help.
+Thank you!
+
+Contributors include:
+
+- Jessica Forde
+- Ruggero Turra
+- Tadej Novak
+- Frank Sauerburger
+- Lars Nielsen
+- Kanishk Kalra
+- Nikolai Hartmann
+- Alex Held
```

Comment:
Could you please add me as `Alexander Held` instead? I'm not exactly consistent with the use myself, but I think it's better to have the full name in more "official" places like this. Thanks!

Reply:
Done!

Reply:
Thanks!